tests/components/backup/test_websocket.py

```diff
@@ -153,25 +154,26 @@ async def test_info_with_errors(
     await setup_backup_integration(
         hass, with_hassio=False, backups={LOCAL_AGENT_ID: [TEST_BACKUP_ABC123]}
     )
-    hass.data[DATA_MANAGER].backup_agents["domain.test"] = BackupAgentTest("test")
+    mock_agent = mock_backup_agent("test")
+    mock_agent.async_list_backups.side_effect = side_effect
+    hass.data[DATA_MANAGER].backup_agents["domain.test"] = mock_agent
```

Comment:
Why is this needed when `setup_backup_integration` now returns the remote agents?

Reply:
My plan is to, in a follow-up, update tests to use `setup_backup_integration` to create remote agents.

Reply:
Ok, good!